packages/angular-cli/tasks/serve-webpack.ts

```diff
@@ -78,6 +79,10 @@ export default Task.extend({
           console.error(err.stack || err);
           if (err.details) { console.error(err.details); }
           reject(err.details);
+        } else {
+          if(commandOptions.open) {
+            opn(`http://${commandOptions.host}:${commandOptions.port}/`); 
```

Comment:
Are there any pros of that url formatter?
I was trying to make it same as [webpack dev server binary](https://github.com/webpack/webpack-dev-server/blob/master/bin/webpack-dev-server.js#L296) do.